packages/jest-phabricator-coverage/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "jest-phabricator-coverage",
+  "version": "17.0.3",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/facebook/jest.git"
+  },
+  "license": "BSD-3-Clause",
+  "main": "index.js"
```

Comment:
can you make sure it uses the same structure as the other packages in this repo? Basically you have to move the source into `src` and then point this to `build/index.js`.